docs/releases/3.2.5.txt

```diff
@@ -24,3 +24,8 @@ Bugfixes
 
 * Fixed a bug in Django 3.2 where a system check would crash on a model with an
   invalid ``app_label`` (:ticket:`32863`).
+
+* Fixed a bug in Django 3.2 where an import ``from django.db import models``
+  would be missing from a migration triggered by a model which wouldn't have
+  fields from ``django.db.models`` and had a custom ``base`` model
+  (:ticket:`32893`).
```

Comment:
It's not a regression (introduced in f25a385a5eb81ee660a82e6dd41e80938672f69c) so it doesn't qualify for a backport. Please remove release notes.

Reply:
Thanks, removed.